src/main/frontend/fs/sync.cljs

```diff
@@ -1159,174 +1159,211 @@
 
 (extend-type RemoteAPI
   IRemoteAPI
-  (<user-info [this] (.<request this "user_info" {}))
+  (<user-info [this]
+    (go
```

Comment:
How about creating a wrapper like `(wrap-id-and-access-token fn` and reusing it in other places?